Data_Structures_Questions.md

```diff
@@ -4,50 +4,94 @@ Answer the following questions for each of the data structures you implemented a
 
 1. What is the runtime complexity of `enqueue`?
 
+Constant, `O(1)`.
+
 2. What is the runtime complexity of `dequeue`?
 
+Constant, `O(1)`.
+
 3. What is the runtime complexity of `len`?
 
+Constant, `O(1)`.
+
 ## Binary Search Tree
 
 1. What is the runtime complexity of `insert`? 
 
+Linear based on the size of the BST, `O(k)`.
+
 2. What is the runtime complexity of `contains`?
 
+Linear based on the size of the BST, `O(k)`.
+
 3. What is the runtime complexity of `get_max`? 
 
+Linear based on the size of the BST, `O(k)`.
+
 ## Heap
 
```

Comment:
heap method runtimes are also O(log n), we need to base this on the recursive calls, not the list, our heap is based off of a binary tree structure. The get max however is constant which the max val is always in the same spot.

Reply:
whoops, i see you corrected this!